bin/test-image

```diff
@@ -84,7 +84,7 @@ if [ "${TYPE}" = "vm" ]; then
     INSTANCES="${TEST_IMAGE}"
 
     # Cloud-init testing.
-    if [ "${VARIANT}" = "cloud" ]; then
+    if [ "${VARIANT}" = "cloud" ] || lxc exec "${name}" -- sh -c "command -v cloud-init"; then
```

Comment:
Missed that one: `${name}` -> `${TEST_IMAGE}`

Also, below is one more occurrence where cloud image is tested (for privileged containers)